consistent-sampling/src/main/java/io/opentelemetry/contrib/samplers/ConsistentAlwaysOffSampler.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.samplers;
+
+import io.opentelemetry.contrib.state.OtelTraceState;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public final class ConsistentAlwaysOffSampler extends ConsistentSampler {
```

Comment:
Our general pattern is to not make the implementations public and expose static factories on the `ConsistentSampler` itself

Reply:
addressed in f7911c35c4c502b205493b02e10e5e441d47a00b